File: src/test/fixtures/typescript/parameterCount.ts
// clean — not flagged by parameter count
function cleanFewParams(a: number, b: number): number {
    return a + b;
}

// flagged — parameter count (medium)
function flaggedManyParams(a: number, b: number, c: number, d: number, e: number, f: number): number {
    return a + b + c + d + e + f;
}

// flagged — parameter count (high)
function flaggedTooManyParams(a: number, b: number, c: number, d: number, e: number, f: number, g: number, h: number, i: number): number {
    return a + b + c + d + e + f + g + h + i;
}

File: src/test/fixtures/csharp/ParameterCount.cs
class FewParameters
{
    static int CleanFewParams(int a, int b) => a + b;
}

class ManyParameters
{
    static int FlaggedManyParams(int a, int b, int c, int d, int e, int f) => a + b + c + d + e + f;
}

class TooManyParameters
{
    static int FlaggedTooManyParams(int a, int b, int c, int d, int e, int f, int g, int h, int i) => a + b + c + d + e + f + g + h + i;
}
